src/utils/common-utils.ts

```diff
@@ -144,51 +118,6 @@ export const getResourceFileType = (
     return ext;
 };
 
-/* Helpers */
```

Comment:
Note: unrelated change, but this massive object is not being used anywhere and should be deleted